### ISSUE TITLE
vcf-merge complains about malformed header

### ISSUE THREAD

**Issue by subwaystation** [apeltzer/EAGER-CLI#22]
Hi,

when I want to merge VCF files generated by `EAGER` using `vcftools'` `vcf-merge`, the following error occures:

```
Use of uninitialized value $attr_value in pattern match (m//) at /usr/lib/perl5/site_perl/Vcf.pm line 2989, <__ANONIO__> line 2.
Use of uninitialized value $attr_value in pattern match (m//) at /usr/lib/perl5/site_perl/Vcf.pm line 2989, <__ANONIO__> line 2.
Could not parse header line: FILTER=<ID='BaseQualityFilter',Description=""QUAL>30\"">
Stopped at [QUAL>30\""].

 at /usr/lib/perl5/site_perl/Vcf.pm line 172, <__ANONIO__> line 2.
	Vcf::throw(Vcf4_2=HASH(0x8654a0), "Could not parse header line: FILTER=<ID='BaseQualityFilter',D"...) called at /usr/lib/perl5/site_perl/Vcf.pm line 2981
	Vcf4_0::parse_header_line(Vcf4_2=HASH(0x8654a0), "##FILTER=<ID='BaseQualityFilter',Description=\"\"QUAL>30\\\"\">\x{a}") called at /usr/lib/perl5/site_perl/Vcf.pm line 625
	VcfReader::_next_header_line(Vcf4_2=HASH(0x8654a0)) called at /usr/lib/perl5/site_perl/Vcf.pm line 598
	VcfReader::parse_header(Vcf4_2=HASH(0x8654a0)) called at /usr/bin/vcf-merge line 183
	main::init_cols(HASH(0xcdf1c0), Vcf4_2=HASH(0xd5f618)) called at /usr/bin/vcf-merge line 279
	main::merge_vcf_files(HASH(0xcdf1c0)) called at /usr/bin/vcf-merge line 12
```

Here is the beginning of the header of one example VCF:

```
##fileformat=VCFv4.2
##FILTER=<ID='BaseQualityFilter',Description=""QUAL>30\"">
##FILTER=<ID='CoverageFilter',Description=""DP>0\"">
##FILTER=<ID=LowQual,Description="Low quality">
##FORMAT=<ID=AD,Number=R,Type=Integer,Description="Allelic depths for the ref and alt alleles in the order listed">
##FORMAT=<ID=DP,Number=1,Type=Integer,Description="Approximate read depth (reads with MQ=255 or with bad mates are filtered)">
##FORMAT=<ID=GQ,Number=1,Type=Integer,Description="Genotype Quality">
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
```

The problem lines are line `2` and `3`. I suppose the string in the `Description` is the problem. Maybe you can fix this in future versions? Or are these headers generated by an external tool?

All the best.

**Comment by apeltzer** [comment]
I have an idea what might have gone wrong here.... 

`
##FILTER=<ID='BaseQualityFilter',Description=""QUAL>30\"">
##FILTER=<ID='CoverageFilter',Description=""DP>0\"">
`

Can you change these to:

`
##FILTER=<ID='BaseQualityFilter',Description="QUAL>30">
##FILTER=<ID='CoverageFilter',Description="DP>0">
`

(remove the extra " and \" ) ? This should fix it.... if you confirm this helps, I will update the filtered output accordingly.

**Comment by subwaystation** [comment]
That did the trick :)

Thanks!

**Comment by apeltzer** [comment]
Bug fixed in 1e886a0e4b7a6483bdf53f3a2d51bd3227f2313c
